ui/docs-app/docs_app/theme/assets/semantic-ui/js/docs_app/forms/deposit/DepositForm.jsx

```diff
@@ -239,9 +241,7 @@ export const DepositForm = () => {
                       placeholder={i18next.t("Choose licenses")}
                       clearable
                       optionsListName="licenses"
-                      helpText={i18next.t(
-                        "If a Creative Commons license is associated with the resource, select the appropriate license option from the menu. We recommend choosing the latest versions, namely 3.0 Czech and 4.0 International."
-                      )}
+                      helpText={i18next.t("Creative Commons license")}
```

Comment:
Why this change? There are possibly multiple licences other than CC

Reply:
@mirekys I only changed the key, as I had some weird issue with key not being found even though it exists in translation.json, so I thought maybe the key was too long or something :) and it worked, but now when I returned the key as full string "If a Creative Commons license is associated with the dataset, select the appropriate license option from the menu. We recommend choosing the latest versions, namely 3.0 Czech and 4.0 International.", it seems to be working also, so I returned it like that. Sorry about that..